test/acs-engine-test/promoteToFailure/promoteToFailure.go

```diff
@@ -0,0 +1,219 @@
+package promoteToFailure
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	azureStorage "github.com/Azure/azure-sdk-for-go/storage"
```

Comment:
ditto with the alias `azureStorage`, let's get rid of the `upperCaseName` thing here

Reply:
fixed